bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md

```diff
@@ -0,0 +1,1150 @@
+Generally, whatever is done is done.
+
+Sjors Provoost: 00:02:41
+
+Yes. And then there are some minor updates throughout the rest of the year.
+That might be 24.1, 24.2, etc.
+And those typically are just bug fixes, but no major changes.
+
+Aaron van Wirdum: 00:02:51
+
+And right now there's a release candidate.
+At the time of recording this podcast, there's I think the first release candidate for Bitcoin Core 24.
+
+## Release Candidate
+
+What does a release candidate mean?
+
+Sjors Provoost: 00:03:01
+
+Kind of what the word suggests, it's a candidate for the release.
+So if all goes well, then the actual release will be made identical to the last candidate.
+But the idea is that people who know what they're doing should be downloading that release candidate and or compiling it themselves and playing around with it and making sure that nothing crashes the things they use especially if you run some sort of automated service like you might be running an exchange or maybe you're running BTC pay or something like that you want to make sure nothing breaks in your setup because Bitcoin Core tries to not break things that use it but it happens occasionally.
+
+Aaron van Wirdum: 00:03:39
+
+Right so if there's no report of anything breaking then this release candidate will basically be the release right?
+
+Sjors Provoost: 00:03:45
+
+Yeah, but I think there are already some things that have been improved so there will definitely be a second release candidate.
+
+Aaron van Wirdum: 00:03:51
+
+okay yeah, and I think in previous Bitcoin core releases there were an average of four or so.
+
+Sjors Provoost: 00:03:56
+
+Yeah, I'd say that's typical.
+So usually there's two weeks between them, So you release it and then maybe after a week or so, people start fixing things, and then after two weeks there is a new one.
+
+Aaron van Wirdum: 00:04:06
+
+Right, so realistically well technically the new Bitcoin Core 24 release could be released any day now, but practically speaking will probably be at least a couple more weeks, right?
+
+Sjors Provoost: 00:04:17
+
+Yeah, I'd say so.
+I think there is a release, attempted release schedule on the Bitcoin Core repository, which I think is for mid-October.
+
+Aaron van Wirdum: 00:04:28
+
+So, there are a bunch of new features as well as bug fixes and improvements, sorry, performance improvements in this new release.
+We're going to discuss a couple of these.
+This is kind of just our selection.
+If this podcast was made by two other people, by Ors and Sharon, they might make a different selection.
+But we, Sjors and Aaron, we made the selection.
+
+Sjors Provoost: 00:04:59
+
+Yeah, I mean, you can look at all the commits, so all the atomic changes that go into a release, and there could be a thousand of them, but most of them are very boring, or at least they are very boring unless you are really into details of how compilers work or into cleaning up code for the sake of cleaning it up.
```

Comment:
Another example of a filler woord/phrase at the beginning of the sentece. 
"Yeah, I mean" don't add significant meaning to the sentence and can be removed.